src/linux/loongarch64/arch_defs_.h

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) 2021 The strace developers.
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#define PERSONALITY0_AUDIT_ARCH { AUDIT_ARCH_LOONGARCH64, 0 }
```

Comment:
Please add AUDIT_ARCH_LOONGARCH64 to src/xlat/audit_arch.in, too.

Reply:
Sure; I'll do so later (after finishing my `$DAY_JOB`).